tests/cli/file.spec.ts

```diff
@@ -0,0 +1,5 @@
+import { test } from '@playwright/test';
+
+test.describe('zkApp-CLI', () => {
+  // TODO: Add sources&tests generation tests
```

Comment:
Should we create an issue and reference it here? I always like having an issue tied to a TODO, makes it more concrete